### PR TITLE
Update web.js

### PR DESCRIPTION
**Pull request by jpdeoliveira**

### CODE REVIEW

File: web.js [hunk 1]
```diff
@@ -12,7 +12,23 @@ addon.webhook('room_message', /@[Dd]orman/, function *() {
...
+    "That is a veiled HR VIOLATION.",
+    "Put $2.00 in the jar.",
+    "I want my $2.00.",
+    "Collect your pink slip at Nicole's desk.",
+    "Prontos?",
+    "Does Evan still work here?",
+    "Is it 4:20 yet?",
+    "It's Jason's fault.",
+    "I'm wearing jeans right now...",
+    "/me is replying...",
```
Comment (aj-may) [inline web.js]: I don't think these are going to work

Comment (jpdeoliveira) [inline web.js]: shiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiittt

Comment (aj-may) [inline web.js]: just yank those last 2

**Comment by aj-may** [comment]
Can you squash your commits.

**Comment by aj-may** [comment]
Screw it. We'll do it in production.

**Pull request merged**